nemo/collections/nlp/data/language_modeling/megatron/gpt_prompt_learning_dataset.py

```diff
@@ -66,6 +66,9 @@ def __init__(
         if isinstance(datasets[0], dict):
             self.load_data(datasets)
 
+        elif isinstance(datasets[0], str) and "taskname" in datasets[0]:
+            self.load_data(datasets)
+
```

Comment:
Add GPTPromptLearningDataset doc string since the datasets can be different types.

Reply:
Will do